pkg/providers/amifamily/bootstrap/windows.go

```diff
@@ -0,0 +1,79 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bootstrap
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"sort"
+	"strings"
+	"sync"
+
+	"github.com/aws/karpenter-core/pkg/apis/v1alpha5"
+
+	"github.com/samber/lo"
+)
+
+type Windows struct {
+	Options
+}
+
+// nolint:gocyclo
+func (w Windows) Script() (string, error) {
+	var userData bytes.Buffer
+	userData.WriteString("<powershell>\n")
+	userData.WriteString("[string]$EKSBootstrapScriptFile = \"$env:ProgramFiles\\Amazon\\EKS\\Start-EKSBootstrap.ps1\"\n")
+	userData.WriteString(fmt.Sprintf(`& $EKSBootstrapScriptFile -EKSClusterName "%s" -APIServerEndpoint "%s"`, w.ClusterName, w.ClusterEndpoint))
+	if w.CABundle != nil {
+		userData.WriteString(fmt.Sprintf(" -Base64ClusterCA \"%s\"", *w.CABundle))
+	}
+	kubeletExtraArgs := strings.Join([]string{w.nodeLabelArg(), w.nodeTaintArg()}, " ")
```

Comment:
Can the `kubeletExtraArgs` here pass the rest of the Provisioner's `spec.kubeletConfiguration` through? For an example of this, you could look at: https://github.com/aws/karpenter/blob/main/pkg/providers/amifamily/bootstrap/eksbootstrap.go#L90

Reply:
Will align with the Linux nodes.